### DISCUSSION TITLE
Refactor dials_regression so it can be made public

### DISCUSSION THREAD

**Discussion by phyy-nx** [dials/dials#1714] [Show and tell]
dials_regression is currently an svn repository with code, data for regression tests, and publication notes.  Since it is hosted on cci.lbl.gov, if a user wants to run the dials_regression tests, they need an account at LBL.  I propose getting it ready for public distribution thusly:

- [ ] 1. Move the doc folder into new repository.  Presumably still svn, and we are happy to host it at lbl.  Idea is that the notes, docs and in-process publication tex files are better served living in a separate location from the test data.  Proposed names: dials_doc, dials_intranet, dials_notes, or something similar.
- [x] 2. Move python code that's run by the nightly tests into dials and dxtbx proper.  There are only 3 tests in run_tests.py anyways.

With these two changes, dials_regression will be a bucket of test data that we can zip it up and host for public downloads like xia2_regression, without worrying about distributing raw notes and extraneous code.

Thoughts?  I'm happy to set this up, and I would do my best to preserve commit histories, if possible.

**Comment by graeme-winter** [comment]
100% behind this - since @Anthchirp did this for xia2_regression already may have thoughts?

**Comment by dagewa** [comment]
I think all relevant tests are now in DIALS or dxtbx. There is still a lot of material in `doc/` though
